src/app/pages/LendingPage/components/LendingDialog/index.tsx

```diff
@@ -290,8 +290,10 @@ export function LendingDialog({
           />
         </div>
       </Dialog>
-      {type === 'add' && <TxDialog tx={lendTx} onSuccess={() => {}} />}
-      {type === 'remove' && <TxDialog tx={unlendTx} onSuccess={() => {}} />}
+      {type === 'add' && <TransactionDialog tx={lendTx} onSuccess={() => {}} />}
```

Comment:
`onSuccess` is an optional prop in `TransactionDialog` so you don't need to provide it. In case you do want to pass an empty function there (I don't think so), use `noop` constant.